light_dark_mode/src/components/Main.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+export default function Main({darkMode}) {
+    const facts = ["Was first released in 2013", "Was originally created by Jordan Walke", "Has well over 100K stars on GitHub", "Is maintained by Facebook", "Powers thousands of enterprise apps, including mobile apps"]
```

Comment:
Please keep max line width 80 characters. It is easier to read. One item per line